docker/dev/docker-compose.yml

```diff
@@ -111,7 +111,7 @@ services:
       CONNECTOR_PORT: 3002
       OPEN_PAYMENTS_PORT: 80
       DATABASE_URL: postgresql://rafiki_backend:rafiki_backend@postgres/rafiki_backend
-      USE_TIGERBEETLE: true
+      USE_TIGERBEETLE: false
```

Comment:
why?

Reply:
well, tigerbeetle was not working on my linux since some weeks ago.

Reply:
it can be changed on your machine, but should remain to true in the repo